client/layout/guided-tours/docs/examples/README.md

```diff
@@ -10,6 +10,6 @@ When we archive an unused tour as an example, we aim to move all the selectors o
 
 - `simple-payments-end-of-year-guide.js`: This tour was used as a temporary end-of-year advertisement for the Simple Payments feature. It triggers on a variety of paths but only for premium and business users. The tour guides people through the creation of a page and the insertion of a Simple Payments button.
 
-## Selectors etc.
+## Selectors etc
```

Comment:
Typically `etc.` does include a `.` at the end of it as it is an abbreviation. Should we ignore these cases?

Reply:
Very good point. The rule https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-heading-punctuation doesn't allow lots of configuration, just a list of characters to ban. The plugin uses `:.`, but maybe we should just use `:`

Reply:
I figured out how we can use something `<!--eslint ignore no-heading-punctuation-->` to disable specific rules for the next line, I'll use that to enable `etc.`